src/controllers/registration.controller.js

```diff
@@ -0,0 +1,37 @@
+const registrationService = require('../services/registration.service')
+
+exports.registerGuest = (req, res) => {
+    user = req.body.user
```

Comment:
I would suggest that user just be req.body. Otherwise we have to create an extra nested layer when making the call from the front end that is not really necessary. The only object that will be sent in the request body for calls to this endpoint is the object for creating the new guest registration. So we don't need to add a 'user' label to it to distinguish it from something else. Does that make sense?

Reply:
Makes sense!